src/dmd/backend/sizecheck.c

```diff
@@ -0,0 +1,51 @@
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Copyright:   Copyright (c) 2000-2017 by Digital Mars, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Source:      $(LINK2 https://github.com/dlang/dmd/blob/master/src/dmd/backend/sizecheck.c, backend/sizecheck.c)
+ */
+
+// Configure the back end (optimizer and code generator)
+
+#include        <stdio.h>
+#include        <ctype.h>
+#include        <string.h>
+#include        <stdlib.h>
+#include        <time.h>
```

Comment:
Are all these includes necessary?

Reply:
It's easy to remove them later, so let's postpone this for the sake for progress.